src/server/endpoints/covidcast_utils/db_sources.csv

```diff
@@ -19,7 +19,7 @@ safegraph,safegraph-weekly,SafeGraph (Weekly),[SafeGraph](https://docs.safegraph
 usa-facts,usa-facts,USAFacts,USAFacts is a nonprofit that aggregates and standardizes a vast array of data from US government agencies. This source reports confirmed COVID-19 cases and deaths.,confirmed_incidence_num,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/usa-facts.html)
 ght,ght,Google Health Trends,"Google Health Trends tracks Google searches on health-related topics. This source includes Google Health Trends API results for overall searcher interest in a set of COVID-19 related terms about anosmia (lack of smell or taste), which emerged as a symptom of the coronavirus. No longer updated after March 8, 2021.",smoothed_search,unknown,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/ght.html)
 google-survey,google-survey,Google Symptom Surveys,"Delphi ran symptom surveys using a Google tool which collects responses through publisher websites, Google's Opinions Reward app, and similar applications. No longer updated after May 15, 2020.",smoothed_cli,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-survey.html)
-indicator-combination,indicator-combination-nmf,Statistical Combination (NMF),"This source provides signals which are statistical combinations of other sources, calculated by Delphi. It is not a primary data source. No longer updated after March 17, 2021.",nmf_day_doc_fbs_ght,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination-inactive.html)
+indicator-combination,indicator-combination-nmf,Statistical Combination (NMF),"This source provides signals which are statistical combinations of other sources, calculated by Delphi. It is not a primary data source. No longer updated after March 17, 2021.",nmf_day_doc_fbs_ght,CC BY,,[API Documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination.html)
```

Comment:
@tinatownes did you put these changes into the gsheet too?

Reply:
Thanks for the reminder! @tinatownes, I'll leave this to you. This needs to be changed on the "Data sources" tab.